### ISSUE TITLE
psutil.NoSuchProcess "no process found with pid" exception occasionally

### ISSUE THREAD

**Issue by jzwinck** [pelson/conda-execute#37]
I am using conda-execute with success on Ubuntu 14.04 and CentOS 7, but from time to time it fails like this:

```
Traceback (most recent call last):
  File "conda-execute", line 9, in <module>
    load_entry_point('conda-execute==0.6.0', 'console_scripts', 'conda-execute')()
  File "conda_execute/execute.py", line 180, in main
    action()
  File "conda_execute/tmpenv.py", line 191, in cleanup_tmp_envs
    for env, env_stats in envs_and_running_pids():
  File "conda_execute/tmpenv.py", line 157, in envs_and_running_pids
    int(psutil.Process(pid).create_time()) == creation_time)
  File "psutil/__init__.py", line 364, in __init__
    self._init(pid)
  File "psutil/__init__.py", line 404, in _init
    raise NoSuchProcess(pid, None, msg)
psutil.NoSuchProcess: psutil.NoSuchProcess no process found with pid 11783
```

I suppose this is some sort of race condition.  It happens with conda-execute 0.4.1 and 0.6.0.

**Comment by pelson** [comment]
Thanks @jzwinck. I went ahead and put a guard in place in #38. Hopefully that should address the issue, but if anything similar comes up just let me know.

Thanks!

P.S. it will be in release v0.8.0.

**Comment by jzwinck** [comment]
@pelson Thank you--I upgraded to 0.8.0 and it solved this issue but created another: #39.  The upgrade of conda-execute required the upgrade of conda itself at the same time, which may be a confounding factor.
